Assets/Scripts/Gameplay/Damage/DamageCalculator.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Gameplay
+{
+    public class DamageCalculator : IDamageCalculator
+    {
+        /// <summary>
+        /// Temporary solution
+        /// </summary>
+        public float Calculate(IAttackInfo attackInfo, IStats stats)
+        {
+            if (!stats.Imunne)
+            {
+                GetDamage(attackInfo);
+            }
+
+            return GetDamage(attackInfo);
```

Comment:
Its mistake, this return  must return zero value

Reply:
Done